code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1127,3 +1128,107 @@
 			return ..()
 
 #undef BROOM_PUSH_LIMIT
+
+/obj/item/twohanded/supermatter  //Supermatter Halberd, used by Oblivion Enforcers
+	name = "supermatter halberd"
+	desc = "The revered weapon of Oblivion Enforcers, used to enforce the Order's will."
```

Comment:
General question about the halberd. Shouldn't this maybe be unable to be dusted by the SM too?

Reply:
yeah, i'll get to this later with the early return thing